packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -130,16 +170,20 @@ export class Builder extends EventEmitter {
     }
 
     try {
-      await this.updateSchema();
-      await this.generateSchemaTypes();
+      await Promise.all(
+        this.getSchemaPaths().map(this.updateSchema.bind(this)),
+      );
     } catch (error) {
       this.emit('error', error);
       return;
     }
 
     try {
       await Promise.all(
-        glob.sync(globs).map(this.updateDocumentForFile.bind(this)),
+        globs
+          .map((pattern) => glob.sync(pattern))
```

Comment:
this globbing assumes the workspace and process cwd are the same, we can override the glob cwd with options.